packages/core-app-api/src/apis/implementations/AppLanguageApi/AppLanguageSelector.ts

```diff
@@ -105,6 +105,17 @@ export class AppLanguageSelector implements AppLanguageApi {
     return { languages: this.#languages.slice() };
   }
 
+  getLanguageDisplayName(language: string): string {
```

Comment:
Hey sorry to trouble you but, would you mind not adding this yet to the interface? If we could revert the core-*-api changes it'd be great, and instead inline the code in the control for now.

The reason is that we are still stabilizing this interface and there's only a single usage point for this need - if we have a couple of needs later on we can consider centralizing it, when we have more knowledge. Could be the case that it should return an object with both native and english labels for example, or an i18n key that you can, in turn, use to look it up second hand ... we just don't know for sure yet.

Reply:
Yep, I can do that, no problem!